runtime/doc/lua.txt

```diff
@@ -3388,6 +3388,34 @@ Iter:find({f})                                                   *Iter:find()*
     Return: ~
         any
 
+Iter:flatten({depth})                                         *Iter:flatten()*
+    Flattens a |list-iterator| pipeline, un-nesting nested values. Errors if
+    it attempts to flatten a dict-like value
+
+    Example:
+
+    >lua
+        vim.iter({1, {2}, {{3}}}):flatten():totable()
+
+    <(returns `{1, 2, {3}}` )
```

Comment:
```suggestion
    -- { 1, 2, {3} }
```
(for example)

Let's make the formatting consistent with others, e.g. `Iter:enumerate()`, `Iter:find()`, etc. Also please use proper whitespaces throughout the examples, as per the standard stylua style.

Reply:
I see this is not fixed yet?

Reply:
sorry, I think that this one didn't apply. I'll manually apply it since the suggestion was edited

Reply:
this is fixed now